conjure-java-client-verifier/src/test/java/com/palantir/conjure/java/compliance/SingleParamServicesTest.java

```diff
@@ -44,42 +44,61 @@
 
     private static final Logger log = LoggerFactory.getLogger(SingleParamServicesTest.class);
     private static final ObjectMapper objectMapper = ObjectMappers.newClientObjectMapper();
-    private static ImmutableMap<String, Object> servicesMaps = ImmutableMap.of(
-            "singlePathParamService", VerificationClients.singlePathParamService(server),
-            "singleHeaderService", VerificationClients.singleHeaderService(server),
-            "singleQueryParamService", VerificationClients.singleQueryParamService(server));
+    private static ImmutableMultimap<String, Object> servicesMaps = ImmutableMultimap.<String, Object>builder()
+            .putAll(
+                    "singlePathParamService",
+                    VerificationClients.singlePathParamService(server),
+                    VerificationClients.dialogueSinglePathParamService(server))
+            .putAll(
+                    "singleHeaderService",
+                    VerificationClients.singleHeaderService(server),
+                    VerificationClients.dialogueSingleHeaderService(server))
+            .putAll(
+                    "singleQueryParamService",
+                    VerificationClients.singleQueryParamService(server),
+                    VerificationClients.dialogueSingleQueryParamService(server))
+            .build();
```

Comment:
could we move this var into the `data()` func so it's clear we're only indexing into the multimap (which is a little gross) to build up the testcases?

Reply:
looks like we actually rely on having the full name: https://github.com/palantir/conjure-java/pull/758/files#diff-99ee67231f23433ff3738d89c722d01dR125 🤦‍♂